packages/lesswrong/components/common/HeadTags.tsx

```diff
@@ -10,14 +10,24 @@ export const faviconUrlSetting = new PublicInstanceSetting<string>('faviconUrl',
 const tabTitleSetting = new PublicInstanceSetting<string>('forumSettings.tabTitle', 'LessWrong', "warning")
 const tabLongTitleSetting = new PublicInstanceSetting<string | null>('forumSettings.tabLongTitle', null, "optional")
 
-const HeadTags = ({ogUrl: ogUrlProp, canonicalUrl: canonicalUrlProp, description: descriptionProp, title: titleProp, image, useSmallImage=false, noIndex}: {
+const HeadTags = ({
+  ogUrl: ogUrlProp,
+  canonicalUrl: canonicalUrlProp,
+  description: descriptionProp,
+  title: titleProp,
+  image,
+  useSmallImage=false,
+  noIndex,
+  structuredData
+}: {
   ogUrl?: string,
   canonicalUrl?: string,
   description?: string|null,
   title?: string,
   image?: string|null,
   useSmallImage?: boolean,
   noIndex?: boolean,
+  structuredData?: any,
```

Comment:
I think we could probably be a bit more specific than `any` here? Like `Record<string, unknown>` maybe?

If not then we should probably use one of the `AnyBecause...` types.